docs/tutorials/Setting-up-Raspberry-Pi-Zero-W.md

```diff
@@ -0,0 +1,153 @@
+
+*by Eric Tang*
+
+This is a tutorial for those of you who really want to run ELL on a Raspberry Pi Zero W. Since Raspberry Pi Zero is based on ARMV6 which has little support from latest Conda, you have to build many python packages in other tutorials by your own from source. You need lots of patience to make it work since Raspberry Pi Zero has much less power compared with a Raspberry Pi 3 Model B. We still recommend you to follow the instructions for [setting up your Raspberry Pi](/ELL/tutorials/Setting-up-your-Raspberry-Pi) for a much easier starting experience.
+
+## Basic Pi Setup
+
+### Power Adapter and Power Cable
+AI workloads guzzle power, so be sure to use a high quality power supply. If you use a USB power adapter and micro USB cable, choose an adapter rated for at least 12 Watts (2.4 Amps) per USB port. We've had good experience with 12W-per-port USB power adapters from Apple, Anker, and Amazon Basics. Long and thin micro USB cables will often result in a noticeable voltage drop and fail to provide sufficient power to the Raspberry Pi. For a few extra dollars you can get a nice name-brand cable, like the Anker PowerLine, and save yourself a lot of frustration.
+
+### Operating System
+Our tutorials assume that the operating system running on your Pi is *Raspbian Jessie* ([NOOBS](https://downloads.raspberrypi.org/NOOBS/images/NOOBS-2017-07-05/) or [image](https://downloads.raspberrypi.org/raspbian/images/raspbian-2017-07-05/)), not the more recent *Raspbian Stretch*.
+
+### Network
+Connect your Pi to the network, either over Wifi or with an Ethernet cable. We will use the network both to download required software to the Pi and to transfer compiled ELL models from your computer to the Pi. Raspberry Pi Zero has a Wifi module on board which makes it easy to connect over Wifi. If you are using a Raspberry Pi Zero with no integrated Wifi module, you may have to connect your board via a Ethernet cable or install a third part wifi dongle.
+
+### CMake
+We use CMake on the Raspberry Pi to create Python modules that can be called from our tutorial code. To install CMake on your Pi, connect to the network, open a terminal window, and type
+
+```shell
+sudo apt-get update
+sudo apt-get install -y cmake
+```
+
+### Python 3.4 installation
+Our tutorials require Python 3.4 on the Pi (and Python 3.6 on your computer). Since the last Conda support for ARMV6 is not actively maintained, here we install python 3.4 directly into our system path. We will also install pip3 which is a python package management tool.
+
+To install python 3.4 on your Raspbian Jessie, type the following:
+
+```shell
+sudo apt-get install python3 python3-dev
+wget https://bootstrap.pypa.io/get-pip.py
+sudo python3 get-pip.py
+```
+
+### OpenBLAS
+OpenBLAS is a library for fast linear algebra operations, which can significantly increase the speed of your models. It is optional, but highly recommended. To install OpenBLAS, type the following.
```

Comment:
Move "To install OpenBLAS, type the following." to line 47, or the end of line 46.

Reply:
Moved over the line 47.